src/content/docs/auto-telemetry-pixie/manage-pixie-data.mdx

```diff
@@ -0,0 +1,98 @@
+ --set global.licenseKey=<NR LICENSE KEY> \
+ --set global.cluster=pixie-auto-telemetry \
+ --namespace=newrelic \
+ --set newrelic-infrastructure.privileged=true \
+ --set ksm.enabled=true \
+ --set prometheus.enabled=true \
+ --set kubeEvents.enabled=true \
+ --set logging.enabled=true \
+ --set newrelic-pixie.enabled=true \
+ --set newrelic-pixie.apiKey=<PIXIE API KEY> \
+ --set pixie-chart.enabled=true \
+ --set pixie-chart.deployKey=<PIXIE DEPLOY KEY> \
+ --set pixie-chart.clusterName=pixie-auto-telemetry \
+ --set newrelic-pixie.excludeNamespacesRegex="default|kube-node-lease|kube-public|kube-system|newrelic|newrelic-custom-metrics|olm|px-operator" \ 
+ --set newrelic-pixie.excludePodsRegex="load-test.*|loadgen.*"
+```
+
+If you're just upgrading an existing install, then this is a much simpler approach:
+
+```
+helm upgrade newrelic-bundle newrelic/nri-bundle --reuse-values -n newrelic --set newrelic-pixie.excludeNamespacesRegex="default|k
+```
+
+Learn more about the available parameters for the `newrelic-pixie` Helm chart [here](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie#chart-details).
+
+
+
+## Use Kubernetes features to collect selected data [#selected-data]
+
+When you deploy Auto-telemetry with Pixie, you're actually enabling the Auto-telemetry with Pixie helm chart, as well as other New Relic components included in the New Relic infrastructure bundle. The [Pixie Edge Module (PEM)](https://docs.pixielabs.ai/tutorials/custom-data/distributed-bpftrace-deployment#background) pods are deployed to the cluster as a Kubernetes DaemonSet, which means that by default, a pod is scheduled to every cluster node and is responsible for collecting all of the observability metrics for that node.
+
+In Kubernetes, you can assign pods to a specific subset of cluster nodes using nodeSelectors, taints/tolerations, and node affinity/anti-affinity. That way, you'll only collect metrics for the nodes you choose, instead of every node. This is helpful if you only want to deploy Auto-telemetry with Pixie to five of your ten cluster nodes, for example. Maybe the designated five nodes are responsible for high-priority workloads, or perhaps you're running both Linux and Windows nodes in your cluster and only want to deploy on the Linux nodes, since Windows nodes are not currently supported. This is now possible to do by providing an additional option to the guided installation command. This option passes an escaped JSON string to the Auto-telemetry with Pixie chart, which enables a nodeSelector that only schedules the PEM DaemonSet on nodes with the label `pixie=allowed`.
+
+```
+--set pixie-chart.patches.vizier-pem='\{\"spec\"\: \{\"template\"\: \{\"spec\"\: \{ \"nodeSelector\"\: \{\"pixie\"\: \"allowed\" \}\}\}\}\}'
+```
+
+If you're using a "values" file, common in the Helm world, then this is what that would look like in the `nri-bundle values.yaml`:
```

Comment:
should values be single. backticks instead of quotes? 
I would say "common in Helm" rather than "common in the Helm world," but you are a much more friendly writer than I am! :-)